extensions/amp-story/1.0/story-requests.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @fileoverview Utils for requests on amp-story and dependencies.
```

Comment:
The only file using the `story-` prefix is `story-analytics`, and really it should be `amp-story-analytics`.

Can you rename this file to something else?

Reply:
`request-utils` sg?